src/messaging/Message.ts

```diff
@@ -222,13 +224,9 @@ export interface IRequestLegitimations extends IMessageBodyBase {
   content: IPartialClaim
   type: MessageBodyType.REQUEST_LEGITIMATIONS
 }
-export interface ISubmitLegitimations extends IMessageBodyBase {
-  content: {
-    claim: IPartialClaim
-    legitimations: IAttestedClaim[]
-    delegationId: DelegationNode['id'] | null
-  }
-  type: MessageBodyType.SUBMIT_LEGITIMATIONS
+export interface ISubmitTerms extends IMessageBodyBase {
```

Comment:
Question:
Just curious, why is `ISubmitLegitimations` replaced alltogether?

Reply:
I will explain in person.